docs/intro/overview/shared-responsibility-model.md

```diff
@@ -1,49 +1,53 @@
-    - Raise limitations of Gruntwork modules as a feature request or a pull request.
-    - Gruntwork does not guarantee any turn-around time on getting features built or PRs reviewed and merged. Gruntwork modules must also be applicable to a wide range of companies, and will be selective about features to be added.
-- Adding additional Infrastructure as Code to customize it for their company.
-- Communicating with AWS to fix account issues and limitations beyond Gruntwork’s control (quotas, account verification, et cetera).
-- For Reference Architecture customers:
-    - Following all provided manual steps in the Reference Architecture documents where automation is not possible.
-    - Extending and customizing Gruntwork Pipelines beyond the basic CI/CD pipeline that Gruntwork has provided to suit their deployment requirements.
-    - Designing and implementing their AWS infrastructure beyond the Reference Architecture.
-        - Understanding and awareness of AWS resource costs for all infrastructure deployed into their AWS accounts ([Knowledge Base #307](https://github.com/gruntwork-io/knowledge-base/discussions/307) for Ref Arch baseline).
-    - Once deployed, maintaining the Reference Architecture to keep it secure and up to date.
-        - Keeping the Reference Architecture secure in accordance with their company needs.
-        - Understanding and accepting the security implications of any changes made to the Reference Architecture.
-        - Monitoring Gruntwork repositories for updates and new releases and applying them as appropriate.
-        - Maintaining all compliance standards after the Reference Architecture has been delivered.
+:::note
+
+The implementation and maintenance of Gruntwork products in AWS is a shared responsibility between Gruntwork and the customer.
+
+:::
+
+## Gruntwork is responsible for:
+
+1. Providing a tested, updated, and richly featured collection of infrastructure code for the customer to use.
+1. Maintaining a healthy Knowledge Base community where other engineers (including Grunts) post & answer questions.
+1. For Pro / Enterprise Support customers: Answering questions via email and Slack.
+1. For Reference Architecture customers:
+    1. Generating the initial Reference Architecture based on our customer’s selections of available configurations. This includes:
+        1. Our implementation of Landing Zone
+        1. A complete sample app with underlying database and caching layer
+        1. The Gruntwork Pipeline for deploying changes to infrastructure
+        1. An overview of how to use the Reference Architecture
+        1. Manual steps the customer needs to take that can only be done by the customer
```

Comment:
I read this section as: 

"Gruntwork is responsible for (For Reference Architecture customers) manual steps the customer needs to take that can only be done by the customer. 

Do we think this was likely intended for the other "The customer is responsible for" section? Or, if not, we should strike as it is confusing. In actuality, the customer is responsible for steps that can only be done by the customer.